linters/tofu/tofu.test.ts

```diff
@@ -0,0 +1,18 @@
+import { linterCheckTest, linterFmtTest } from "tests";
+import { TrunkLintDriver } from "tests/driver";
+
+// Due to tofu's validate subcommand being disabled by default, we need to manually enable it in our test's trunk.yaml.
+const preCheck = (driver: TrunkLintDriver) => {
+  const trunkYamlPath = ".trunk/trunk.yaml";
+  const currentContents = driver.readFile(trunkYamlPath);
+  const sqlfluffRegex = /- tofu@(.+)\n/;
```

Comment:
nit: this should be renamed (in the terraform test too 🙃 )

Reply:
We have 14 instances of this, will break into its own PR :)